### PR TITLE
add agricultural protection areas data page

### PR DESCRIPTION
**Pull request by gregbunce**

### CODE REVIEW

**Comment by gregbunce** [comment]
see what ya think.

note: i don't know anything about this layer other than the info i received from the steward.

**Comment by gregbunce** [comment]
oops, added one last thing, otherwise, yup, let's merge it!

File: data/farming/agricultural-protection-areas/index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  <figcaption class="caption__text">Agricultural Protection Areas Sample</figcaption>
+</figure>
+{% include abstract.html
+name="Agricultural Protection Areas"
+stewards="UDAF"
+abstract="These datasets contain statewide Agricultural Protection Area boundaries for use in
+planning and land management." %}
+<div class="grid package">
+  <div class="grid__col grid__col--12-of-12">
+    <h3 id="GrazingImprovementAreas">Agricultural Protection Areas</h3>
```
[the remark this answers is not in the document]
Comment (steveoh) [reply data/farming/agricultural-protection-areas/index.html:28]: you marked this as resolved but didn't fix it?

[the remark this answers is not in the document]
Comment (gregbunce) [reply data/farming/agricultural-protection-areas/index.html:28]: oops.  i'm on it.

**Pull request merged**